docs/source/tutorials/code_snippets/interactive.py

```diff
@@ -1,6 +1,5 @@
-from traits import *
-from pyface.api import GUI
-from traitsui import View, Item, ButtonEditor
+from traits.api import *
```

Comment:
nit : instead of doing import *, it will be better to only import the things that are needed/used.